scripts/async_profiler_build_shared.sh

```diff
@@ -5,12 +5,27 @@
 #
 set -euo pipefail
 
-VERSION=v2.8.3g2
-GIT_REV="ef1d3b3c29e7337d52f54035a12ae9a90eb1c710"
+VERSION=v2.8.3g3
+GIT_REV="008a8cf435bedce8eec3ef36fdc67e742b070566"
 
 git clone --depth 1 -b "$VERSION" https://github.com/Granulate/async-profiler.git && cd async-profiler && git reset --hard "$GIT_REV"
 source "$1"
 make all
 
 # add a version file to the build directory
 echo -n "$VERSION" > build/async-profiler-version
+
+# build verifications regarding libstdc++.
```

Comment:
Can u copy the comment from the PR description?
> avoids -static-libstdc++ on glibc; it's required for musl-based because musl-based JVMs aren't always dependent on libstdc++; and we've never seen that with glibc.

Reply:
Kk will add